packages/ketcher-react/src/script/ui/views/modal/components/Text/Text.jsx

```diff
@@ -0,0 +1,46 @@
+/****************************************************************************
+ * Copyright 2020 EPAM Systems
```

Comment:
please update copyright

Reply:
When reviewing form validation with Pavel Sidorenko, we discovered that for many forms instead of actual validation process we pass 'valid: true', which results in the 'OK' button in such forms being constantly enabled. As such, we agreed to add a TODO in order to add a proper validation in the future.

Reply:
fixed